fastcrypto-tbls/src/dkg.rs

```diff
@@ -153,6 +155,9 @@ where
         // to reduce communication.
         let vss_sk = PrivatePoly::<G>::rand(t - 1, rng);
 
+        debug!("Creating party {my_id}, {enc_pk:?} with threshold {t}, {random_oracle:?}");
+        nodes.iter().for_each(|n| debug!("{n:?}"));
```

Comment:
optional for some of these - if you want to avoid iterating or doing other work unless logging is turned on you could use

```
if tracing::enabled!(tracing::Level::DEBUG) { ... }
```